### ISSUE TITLE
Error when attempting to view bank.

### ISSUE THREAD

**Issue by Kream** [AdiAddons/AdiBags#725]
**Describe the bug**
Opening the bank generates the following error. However, the items in the bank are indeed shown.  
```Message: Interface/AddOns/AdiBags/widgets/ItemButton.lua:371: attempt to call upvalue 'ContainerFrame_UpdateCooldown' (a nil value)
Time: Wed Oct 26 22:41:47 2022
Count: 28
Stack: Interface/AddOns/AdiBags/widgets/ItemButton.lua:371: attempt to call upvalue 'ContainerFrame_UpdateCooldown' (a nil value)
[string "@Interface/AddOns/AdiBags/widgets/ItemButton.lua"]:371: in function `UpdateCooldown'
[string "@Interface/AddOns/AdiBags/widgets/ItemButton.lua"]:325: in function `Update'
[string "@Interface/AddOns/AdiBags/widgets/ItemButton.lua"]:303: in function `FullUpdate'
[string "@Interface/AddOns/AdiBags/widgets/ItemButton.lua"]:268: in function <Interface/AddOns/AdiBags/widgets/ItemButton.lua:256>
[string "=[C]"]: in function `Show'
[string "@Interface/AddOns/AdiBags/widgets/ContainerFrame.lua"]:1034: in function `PrepareSections'
[string "@Interface/AddOns/AdiBags/widgets/ContainerFrame.lua"]:1155: in function `FullUpdate'
[string "@Interface/AddOns/AdiBags/widgets/ContainerFrame.lua"]:599: in function <Interface/AddOns/AdiBags/widgets/ContainerFrame.lua:585>
[string "=[C]"]: ?
[string "@Interface/AddOns/AdiBags/core/Utility.lua"]:100: in function <Interface/AddOns/AdiBags/core/Utility.lua:92>
[string "@Interface/AddOns/AdiBags/widgets/LayeredRegion.lua"]:119: in function <Interface/AddOns/AdiBags/widgets/LayeredRegion.lua:108>

Locals: self = AdiBagsBankItemButton19 {
 0 = <userdata>
 bag = -1
 IconOverlay2 = Texture {
 }
 OnUpdateItemContextMatching = <function> defined @Interface/FrameXML/ItemButtonTemplate.lua:632
 inventorySlot = 82
 IconOverlay = Texture {
 }
 itemId = 168523
 SetAlpha = <function> defined @Interface/FrameXML/ItemButtonTemplate.lua:616
 section = AdiBagsSection31 {
 }
 filterName = "Item category"
 IconBorder = Texture {
 }
 bagFamily = 0
 GetItemLocation = <function> defined @Interface/FrameXML/ItemButtonTemplate.lua:604
 icon = AdiBagsBankItemButton19IconTexture {
 }
 container = AdiBagsContainer2 {
 }
 SetItemButtonTexture = <function> defined @Interface/FrameXML/ItemButtonTemplate.lua:655
 acquired = true
 UpdateTooltip = <function> defined @Interface/FrameXML/BankFrame.lua:59
 GetBagID = <function> defined @Interface/FrameXML/BankFrame.lua:151
 SetItem = <function> defined @Interface/FrameXML/ItemButtonTemplate.lua:538
 texture = 1336885
 showMatchHighlight = true
 OnItemContextChanged = <function> defined @Interface/FrameXML/ItemButtonTemplate.lua:404
 itemLink = "[Ub3r Module: Scrap Cannon]"
 SetItemButtonQuality = <function> defined @Interface/FrameXML/ItemButtonTemplate.lua:643
 count = 1
 UpdateItemContextMatching = <function> defined @Interface/FrameXML/ItemButtonTemplate.lua:444
 SetItemSource = <function> defined @Interface/FrameXML/ItemButtonTemplate.lua:522
 SetItemButtonCount = <function> defined @Interface/FrameXML/ItemButtonTemplate.lua:608
 Cooldown = AdiBagsBankItemButton19Cooldown {
 }
 SetItemInternal = <function> defined @Interface/FrameXML/ItemButtonTemplate.lua:543
 IconTexture = AdiBagsBankItemButton19IconTexture {
 }
 UpdateCraftedProfessionsQualityShown = <function> defined @Interface/FrameXML/ItemButtonTemplate.lua:455
 itemContextMatchResult = 3
 NormalTexture = AdiBagsBankItemButton19NormalTexture {
 }
 GetItemButtonBackgroundTexture = <function> defined @Interface/FrameXML/ItemButtonTemplate.lua:663
 ItemContextOverlay = Texture {
 }
 PostOnShow = <function> defined @Interface/FrameXML/ItemButtonTemplate.lua:408
 UpdateItemContextOverlayTextures = <function> defined @Interface/FrameXML/ItemButtonTemplate.lua:493
 GetItemID = <function> defined @Interface/FrameXML/ItemButtonTemplate.lua:585
 GetItemButtonCount = <function> defined @Interface/FrameXML/ItemButtonTemplate.lua:612
 PostOnHide = <function> defined @Interface/FrameXML/ItemButtonTemplate.lua:417
 RegisterBagButtonUpdateItemContextMatching = <function> defined @Interface/FrameXML/ItemButtonTemplate.lua:638
 IconQuestTexture = Texture {
 }
 hasItem = true
 GetItemContextOverlayMode = <function> defined @Interface/Fram
```

**To Reproduce**
1. Interact with a banker

**Expected behavior**
The bank pane should show without any errors. 

**Screenshots**

**World of Warcraft Release and Version (Retail 9.2.7, etc)**
 - Retail 10.0.0 (46313) 

**AdiBags Version**
- v1.9.39

**Have you tried disabling all your other addons to ensure only this addon causes the issue?**
 - Yes

**Additional context**

**Comment by Cidan** [comment]
Thanks for the bug report -- I'll take a look at this tonight and get a fix out ASAP.

**Comment by Cidan** [comment]
Fixed in #727 -- new release going out shortly.
